packages/accounts-base/accounts_common.js

```diff
@@ -66,6 +74,27 @@ export class AccountsCommon {
       const { ServiceConfiguration } = Package['service-configuration'];
       this.loginServiceConfiguration = ServiceConfiguration.configurations;
       this.ConfigError = ServiceConfiguration.ConfigError;
+
+      if(Meteor.settings?.packages?.accounts) {
```

Comment:
You could get the value first and then later test it instead of testing and later getting it.